templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -900,10 +900,7 @@
                                         {% else %}
                                             {{  fileupload(
                                                 "r_pictogram",
-                                                '<p class="lbl__hint">'
-                                                ~ "explanation.procedure.pictogram.missing"|trans
-                                                ~ "explanation.procedure.pictogram.dimensions"|trans
-                                                ~ '</p>',
+                                                "hide",
```

Comment:
Shouldn't the label text be replaced with the tooltip text, and the tooltip be removed? (That's how I understand the ticket)

Reply:
you understand it correctly :D https://github.com/demos-europe/demosplan-core/pull/4197/commits/c077ad4cc84c34b6d6e6e0c105984f1619a34427